File: common/src/main/java/com/wdz/common/base/room/DatabaseOperationListener.java
package com.wdz.common.base.room;

import java.util.List;

/**
 * @Author dezhi.wang
 * @Date 2021/2/1 17:24
 */
public interface DatabaseOperationListener<T> {

    /**
     * 操作成功
     * @param items
     */
    void onSuccess(List<T> items);

    /**
     * 操作失败
     */
    void onFailure();
}

File: kt_common/src/main/java/com/wdz/ktcommon/room/base/DatabaseOperationListener.kt
package com.wdz.ktcommon.room.base

/**
 * @Author dezhi.wang
 * @Date 2021/2/1 17:24
 */
interface DatabaseOperationListener<T> {
    /**
     * 操作成功
     * @param items
     */
    fun onSuccess(items: List<T>?)

    /**
     * 操作失败
     */
    fun onFailure()
}